lms/djangoapps/organizations/views.py

```diff
@@ -152,10 +167,19 @@ def users(self, request, pk):
     def groups(self, request, pk):
         """
         Add a Group to a organization or retrieve list of groups in organization
+        - GET: Returns groups in an organization
+            * view parameter can be used to get a particular data .i.e. view=ids to
+            * get list of group ids
+
         """
         if request.method == 'GET':
             group_type = request.QUERY_PARAMS.get('type', None)
+            view = request.QUERY_PARAMS.get('view', None)
             groups = Group.objects.filter(organizations=pk)
+            if view == 'ids':
```

Comment:
It seems to me this view formatter should be located below the group_type filter on line 183.